hack/generator/pkg/astbuilder/assignments.go

```diff
@@ -42,3 +42,50 @@ func SimpleAssignmentWithErr(lhs dst.Expr, tok token.Token, rhs dst.Expr) *dst.A
 		},
 	}
 }
+
+// SimpleIf creates a simple if statement with a single statement in each branch
+//
+// if <condition> {
+//     <trueBranch>
+// } else {
+//     <falseBranch>
+// }
+//
+func SimpleIf(condition dst.Expr, trueBranch dst.Stmt, falseBranch dst.Stmt) *dst.IfStmt {
+	result := &dst.IfStmt{
+		Cond: condition,
+		Body: StatementBlock(trueBranch),
+		Else: StatementBlock(falseBranch),
+	}
+
+	if block, ok := trueBranch.(*dst.BlockStmt); ok {
```

Comment:
How about adding an `EnsureStatementBlock` function that wraps a Stmt if it's not already a BlockStmt? Then you can just call that in the IfStmt literal.

Reply:
Done.